x/wasm/keeper/msg_server_integration_test.go

```diff
@@ -374,11 +428,54 @@ func TestExecuteContract(t *testing.T) {
 	require.NoError(t, wasmApp.AppCodec().Unmarshal(rsp.Data, &instantiateResponse))
 
 	specs := map[string]struct {
-		addr   string
-		expErr bool
+		addr      string
+		expEvents []abci.Event
+		expErr    bool
 	}{
 		"address can execute a contract": {
-			addr:   myAddress.String(),
+			addr: myAddress.String(),
+			expEvents: []abci.Event{
+				createMsgEvent(myAddress), {
+					Type: "execute",
+					Attributes: []abci.EventAttribute{
+						{
+							Key:   []byte("_contract_address"),
+							Value: []byte(instantiateResponse.Address),
+							Index: false,
+						},
+					},
+				}, {
+					Type: "wasm",
+					Attributes: []abci.EventAttribute{
+						{
+							Key:   []byte("_contract_address"),
+							Value: []byte(instantiateResponse.Address),
+							Index: false,
+						}, {
+							Key:   []byte("action"),
+							Value: []byte("release"),
+							Index: false,
+						}, {
+							Key:   []byte("destination"),
+							Value: []byte("link1cvk5jz4jank96cmfrxhf5nn4dmj6atu833yvjq"),
```

Comment:
I think you should delete this line, because it will changed inL508

Reply:
Thanks! I think so, too.